### DISCUSSION TITLE
Is there way to automatically send notification when a long running command completes

### DISCUSSION THREAD

**Discussion by nasen23** [wezterm/wezterm#4109] [Q&A]
Like this functionality in iTerm2 
https://stackoverflow.com/questions/30016716/how-do-i-make-iterm-terminal-notify-me-when-a-job-process-is-complete

**Comment by DevTGhosh** [comment]
Did anyone find an answer to this?

**Reply by leblancfg** [reply]
It's just a special character that you can `printf` from terminal commands:

```console
$ ./my_job && printf "\e]777;notify;%s;%s\e\\" "title" "body"
```
